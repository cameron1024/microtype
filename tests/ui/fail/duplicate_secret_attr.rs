fn main() {}

microtype::microtype! {
    #[secret]
    #[secret]
    String {
        Password
    }
}
