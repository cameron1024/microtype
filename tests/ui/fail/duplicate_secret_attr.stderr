error: macros that expand to items must be delimited with braces or followed by a semicolon
 --> tests/ui/fail/duplicate_secret_attr.rs:5:5
  |
5 |     #[secret]
  |     ^
  |
help: change the delimiters to curly braces
  |
5 |     {}[secret]
  |     ~
help: add a semicolon
  |
5 |     #;[secret]
  |      +

error: duplicate `secret` attribute found
 --> tests/ui/fail/duplicate_secret_attr.rs:5:5
  |
5 |     #[secret]
  |     ^
